docs/book/v4/getting-started/features.md

```diff
@@ -30,45 +30,32 @@ With Mezzio, you can build middleware applications such as the following:
-  with an "Allow" HTTP header containing allowed HTTP methods for invalid
-  requests.
+Stratigility provides limited, literal route matching via its `PathMiddlewareDecorator`.
+Mezzio, however, allows you to utilize dynamic routing capabilities from a variety of router packages, providing much more fine-grained matching capabilities.
+Its routing layer also allows restricting matched routes to specific HTTP methods, and will return a [405 Not Allowed][405-not-allowed-url] response with an [Allow HTTP header][allow-http-header-url] containing the allowed HTTP methods for invalid requests.
 
-  Routing is abstracted in Mezzio, allowing the developer to choose the
-  routing library that best fits the project needs. By default, we provide
-  wrappers for Aura.Router, FastRoute, and the laminas-router.
+Routing is abstracted in Mezzio, allowing the developer to choose the routing library that best fits the project needs.
+By default, we provide wrappers for [Aura.Router][aura-router-url], [FastRoute][fastroute-url], and the [laminas-router][laminas-router-url].
 
-- **PSR-11 Container**
+### A PSR-11 Container
 
-  Mezzio encourages the use of Dependency Injection, and defines its
-  `Application` class to compose a [PSR-11](https://www.php-fig.org/psr/psr-11)
-  `ContainerInterface` instance. The container is used to lazy-load middleware,
-  whether it is piped (Stratigility interface) or routed (Mezzio).
+Mezzio encourages the use of dependency injection, and defines its core `Application` class to compose a [PSR-11][psr11-url] `ContainerInterface` instance.
+The container is used to lazy-load middleware, whether it is piped (Stratigility interface) or routed (Mezzio).
 
-- **Templating**
+### Flexible Templating
 
-  While Mezzio does not assume templating is being used, it provides a
-  templating abstraction. Developers can write middleware that typehints on
-  this abstraction, and assume that the underlying adapter will provide
-  layout support and namespaced template support.
+While Mezzio does not assume templating is being used, it provides a templating abstraction.
+Developers can write middleware that typehints on this abstraction, and assume that the underlying adapter will provide layout support and namespaced template support.
 
-- **Error Handling**
+### Error Handling
 
-  Applications should handle errors gracefully, but also handle them differently
-  in development versus production. Mezzio provides both basic error
-  handling via Stratigility's own `ErrorHandler` implementation, providing
-  specialized error response generators that can perform templating or use
-  Whoops.
+Applications should handle errors gracefully, but also handle them differently in development versus production.
+Mezzio provides both basic error handling via Stratigility's own `ErrorHandler` implementation, providing specialized error response generators that can perform templating or use [Whoops][whoops-url].
```

Comment:
Might be worth mentioning the problem-details lib here which solves error handling for apis in a consistent way

Reply:
I'll come back to this in a future PR, if that's okay.